htdocs/request_account/process_new_account.php

```diff
@@ -41,8 +41,8 @@
 $config = NDB_Config::singleton();
 $DB     = Database::singleton();
 
-$res = array();
-$DB->select("SELECT Name, CenterID FROM psc", $res);
+$res       = array();
```

Comment:
I will hold off on this for this PR

I did add the comment to https://github.com/aces/Loris/pull/2961 which is cleaning up this particular class